packages/lesswrong/components/ea-forum/users/EAUsersProfile.tsx

```diff
@@ -630,7 +633,10 @@ const EAUsersProfile = ({terms, slug, classes}: {
             currentIncludeEvents={currentIncludeEvents}
           />}
           <AnalyticsContext listContext="userPagePosts">
-            <PostsList2 terms={postTerms} boxShadow={false} hideAuthor />
+            <HideRepeatedPostsProvider>
+              <ProfileShortform user={user} />
+              <PostsList2 terms={postTerms} boxShadow={false} hideAuthor />
```

Comment:
Ooh, edge case I've just thought of. Is it actually guaranteed that <ProfileShortform> is rendered first, and if not wouldn't the ProfileShortform version be hidden? Since both components do an independent graphql query I think this could happen and it would just _usually_ be the case that the shortform is returned first because it's only fetching one thing

Reply:
I just tested this by adding a setTimeout in ProfileShortform and it is indeed a real issue